src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ModelBinderProviderCollection.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Represents a collection of model binder providers.
+    /// </summary>
+    public class ModelBinderProviderCollection : Collection<IModelBinderProvider>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ModelBinderProviderCollection"/> class that is empty.
+        /// </summary>
+        public ModelBinderProviderCollection()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ModelBinderProviderCollection"/> class
+        /// as a wrapper for the specified list.
+        /// </summary>
+        /// <param name="modelBinderProviders">The list that is wrapped by the new collection.</param>
+        public ModelBinderProviderCollection(IList<IModelBinderProvider> modelBinderProviders)
```

Comment:
I've implemented these constructors according to [`FormatterCollection`](https://github.com/aspnet/Mvc/blob/7166dfecd75779fae4d03bd9b2ff5ed00cfede6e/src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/FormatterCollection.cs), do we need them always?